src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,38 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.controllers;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+public class TelevisionsController {
+    @GetMapping("/televisions")
```

Comment:
Als al je Mapping beginnen met "/television", is het makkelijk om dat niet in elke Mapping opnieuw te schrijven, maar om dan juist een `@`RequestMapping boven aan je klasse te zetten waarin je "/televisions" zet. Dat wordt dan het base pad, in je individuele Mappings kun je daar dan nog aan toeveogen, zoals "/{id}" (of helemaal niks toevoegen)